cmd/crictl/container.go

```diff
@@ -524,6 +536,7 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 	if err != nil {
 		return err
 	}
+	sort.Sort(containerByName(r.Containers))
```

Comment:
Seems docker ps sorts output via createdAt , could you also sort the output this way?

Reply:
Of course. done.

Reply:
I have not studied the docker source code.

Reply:
The name should be `containerByCreated` now.

Reply:
done